src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,150 @@
+           :size            :paragraph-1
+           :weight          :medium
+           :ellipsize-mode  :middle
+           :number-of-lines 1}
+          profile-qr-url]]]
+       [rn/view {:style style/share-button-container}
+        [quo/button
+         {:icon                true
+          :type                :blur-bg
+          :size                32
+          :accessibility-label :link-to-profile
+          :override-theme      :dark
+          :on-press            #(list-selection/open-share {:message profile-qr-url})}
+         :i/share]]]]
+
+     [rn/view {:style style/emoji-hash-container}
+      [rn/view {:style style/profile-address-container}
+       [rn/view {:style style/profile-address-column}
+        [quo/text
+         {:size   :paragraph-2
+          :weight :medium
+          :style  style/emoji-hash-label}
+         (i18n/label :t/emoji-hash)]
+        [rn/touchable-highlight
+         {:active-opacity   1
+          :underlay-color   colors/neutral-80-opa-1-blur
+          :background-color :transparent
+          :on-press         #(rf/dispatch [:share/copy-text-and-show-toast emoji-hash
+                                           (i18n/label :t/emoji-hash-copied)])
+          :on-long-press    #(rf/dispatch [:share/copy-text-and-show-toast emoji-hash
+                                           (i18n/label :t/emoji-hash-copied)])}
+         [rn/text {:style style/emoji-hash-content} emoji-hash]]]]
+      [rn/view {:style style/emoji-share-button-container}
+       [quo/button
+        {:icon                true
+         :type                :blur-bg
+         :size                32
+         :accessibility-label :link-to-profile
+         :override-theme      :dark
+         :style               {:margin-right 12}
+         :on-press            #(rf/dispatch [:share/copy-text-and-show-toast emoji-hash
+                                             (i18n/label :t/emoji-hash-copied)])
+         :on-long-press       #(rf/dispatch [:share/copy-text-and-show-toast emoji-hash
+                                             (i18n/label :t/emoji-hash-copied)])}
+        :i/copy]]]]))
+
+(defn wallet-tab
+  []
+  [rn/text {:style {:color colors/white :text-align :center}} "not implemented"])
+
+(defn view
+  []
+  (let [selected-tab (reagent/atom profile-tab-id)]
+    (fn []
+      [safe-area/consumer
+       (fn [{:keys [top bottom]}]
+         (let [window-width (rf/sub [:dimensions/window-width])]
+           [rn/view {:style (style/screen-container window-width top bottom)}
+            [blur/view style/blur]
+            [header]
+            [rn/view {:style style/tabs-container}
+             [quo/segmented-control
+              {:size                28
+               :scrollable?         true
+               :blur?               true
+               :override-theme      :dark
+               :style               style/tabs
+               :fade-end-percentage 0.79
```

Comment:
I think many of the props here are not really being passed down to the `quo/tab` component and should be removed. Please, take a look https://github.com/status-im/status-mobile/blob/develop/src/quo2/components/tabs/segmented_tab.cljs#L19

Reply:
fixed here : https://github.com/status-im/status-mobile/pull/15528/commits/724e0e1a5964f565049f5a544683cb061fd90d23